microsoft-edge/webview2/concepts/distribution.md

```diff
@@ -3,29 +3,82 @@ description: Distribution options when releasing an app using Microsoft Edge Web
+
+> [!IMPORTANT]
+> Do not ship WebView2 applications in production during the preview.  
+
+The WebView team recommends the following workflow to ensure the Evergreen WebView2 Runtime is available.  
+
+1.  Download the latest [Evergreen WebView2 Runtime Installer][Webview2Installer].  
+1.  Include the installer in your application installer or updater.  
+1.  During your application installation or update, check if the Evergreen WebView2 Runtime is already installed on the user machine.  If not, the application invokes the installer to install the runtime.  
+
+Depending on your scenario, you may need to change the above workflow.  For example, your application installer may download the Evergreen WebView2 Runtime Installer instead of including it in your application package.  
+
+> [!NOTE]
+> Both the WebView2 Runtime and WebView2 Runtime installer are in preview.  The preview has a limited initial scope and is only available as a standalone, per-machine install on Windows 10 on x64.  In the future, support for Windows 7, x86, and ARM64 is planned.  
+
+### Best practices for using WebView2 Runtime and non-stable Microsoft Edge channels  
+
+Consider the following recommendations during the preview.  
+
+1.  Make sure to use the [Evergreen WebView2 Runtime and Installer ][Webview2Installer] to develop or test your packaging and distribution pipeline.  In the future, your production application must include the installer.  
+1.  For developing your application, you may use the Evergreen WebView2 Runtime.  However, as the runtime shifts from the Dev Channel to the Beta Channel or Stable Channel, the runtime build number may not meet the most recent preview WebView2 SDK minimum version requirements.  If you wish to use the most recent SDK, install the Microsoft Edge Canary channel to ensure a compatible build is available on the device.  For more information about versioning, see [Versioning][ConceptsVersioning].  
+1.  To test your web content for compatibility with changes to the platform not available in the Stable Channel, use the appropriate non-Stable Channel as required.  
+
+## Fixed version distribution mode  
+
+> [!NOTE]
+> The fixed version distribution model is currently not available.  
+
+For constrained environments, the WebView team plans to support a fixed version \(previously called bring-your-own\) distribution model.  The fixed version distribution model has the following benefits and limitations.  
+
+*   Allows you to select and package a specific version of the WebView2 Runtime in your application installer.  
```

Comment:
Can we use original wording in the doc? There are a lot of overlap in between bullets and this is not how I would describe benefits/limitations.

Reply:
Updated.
```md
## Fixed version distribution mode  

> [!NOTE]
> The fixed version distribution model is currently not available.  

For constrained environments, there are plans to support a fixed version \(previously called bring-your-own\) distribution mode.  The fixed version distribution mode allows you to select and package a specific version of the WebView2 Runtime.  The fixed version distribution mode allows you to control which version of the WebView2 Runtime is used by your application, and when user machines are updated.  The fixed version distribution mode does not receive any automatic updates, and you should plan to manually apply updates.  
```